util/exec.go

```diff
@@ -23,6 +23,7 @@ import (
 // from the binary's stdout and stderr to this program's stdout and stderr.
 func Exec(bin string, args ...string) error {
 	cmd := exec.Command(bin, args...)
```

Comment:
@dgonyeo general theme here is that acbuild should be as hand-holdy as possible, so prefer to wrap errors and provide suggestions